### PR TITLE
Fixes #8466: Technique userManagement, home directory is not taken into account

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/8466

### CODE REVIEW

**Comment by ncharles** [comment]
Commit modified

File: techniques/systemSettings/userManagement/userManagement/7.1/userManagement.st [hunk 1]
```diff
@@ -129,7 +138,20 @@ bundle agent check_usergroup_user_parameters
...
+      # If we ask to personnalize home, but not define it, it is invalid
+      "usermanagement_user_home_pershome_invalid_${usergroup_user_index}" expression => "usermanagement_user_pershome_${usergroup_user_index}.!usermanagement_user_custom_home_defined_${usergroup_user_index}";
+
+      # Asked to move the home directory
+      "usermanagement_user_custom_home_move_${usergroup_user_index}" expression => strcmp("${usergroup_user_home_move[${usergroup_user_index}]}","true");
+
+      # The request to move home is valid: the path to move to is defined, and we asked to personalize
+      "usermanagement_user_custom_home_move_valid_${usergroup_user_index}" expression => "usermanagement_user_custom_home_move_${usergroup_user_index}.usermanagement_user_custom_home_defined_${usergroup_user_index}.usermanagement_user_pershome_${usergroup_user_index}";
+
+      "usermanagement_user_custom_home_move_invalid_${usergroup_user_index}" expression => "usermanagement_user_custom_home_move_${usergroup_user_index}.!(usermanagement_user_custom_home_defined_${usergroup_user_index}.usermanagement_user_pershome_${usergroup_user_index})";
```
Comment (peckpeck) [inline techniques/systemSettings/userManagement/userManagement/7.1/userManagement.st]: This class is never used

**Comment by peckpeck** [comment]
This is good

**Comment by ncharles** [comment]
Commit modified

**Pull request merged**